extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzRunTimeConfig.java

```diff
@@ -4,8 +4,8 @@
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
 
-@ConfigRoot(phase = ConfigPhase.RUN_TIME)
-public class QuartzRuntimeConfig {
+@ConfigRoot(phase = ConfigPhase.RUN_TIME, name = "quartz")
+public class QuartzRunTimeConfig {
```

Comment:
Was this change really needed? And why do you define a name? Don't we use Runtime elsewhere? And isn't it automatically stripped when generating the name of the config root?

Reply:
> Was this change really needed? And why do you define a name? Don't we use Runtime elsewhere? 

> And isn't it automatically stripped when generating the name of the config root?

Without providing the name I ended up with a `quarkus.quartz-runtime` root, likewise for the buildtime config root.